sig-apps/annual-report-2021.md

```diff
@@ -0,0 +1,60 @@
+# Operational
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate? have a CONTRIBUTING.md file?  
+Yes, both the [README.md](https://github.com/kubernetes/community/blob/master/sig-apps/README.md)
+and the [CONTRIBUTING.md](https://github.com/kubernetes/community/blob/master/sig-apps/CONTRIBUTING.md)
+are up-to-date.
+
+All subprojects correctly mapped and listed in sigs.yaml?  
+Yes
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?  
+We hold small and active biweekly meeting. The call is divided into few sections, such as important announcements, then demos and finally discussions on the current topics. If time allows, we try to review issues and pull requests. All of the meetings are recorded and available online, all the meeting invites are up-to-date and present in community calendar.
+
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?  
+Every subproject is free to provide an update at our biweekly meeting, but we don’t enforce that specifically. OWNERS files are up-to-date.
+
+When was your last monthly community-wide update? (provide link to deck and/or recording)  
+Our last update presentation was on June 18, 2020
+- [Slides](https://docs.google.com/presentation/d/18UcJQs3ThW6Vdgl_mdc1984uU16GwuIhuD0Pujl42xU/edit#slide=id.g401c104a3c_0_0)
+- [Recording](https://youtu.be/ObqQxRRl9RQ?t=1111)
+- [KubeCon NA 2019](https://www.youtube.com/watch?v=uqYStFxu578)
+
+# Membership
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?  
+Yes, all the lists are up-to-date.
+
+How do you measure membership? By mailing list members, OWNERs, or something else?  
+Reviewers and approvers are measured through repository OWNERS file. Membership is measured through mailing list and zoom calls.
+
+How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?  
+We periodically remove inactive reviewers and approvers, invite new contributors we have seen active in to join the reviewers. The bar to be accepted as an approver is significantly higher since we need to ensure the stability of the core controllers and the entire project.
+
+Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?  
+Recently we reviewed and unified the sig-apps-approvers alias which holds the approvers, similarly sig-apps-reviewers was reviewed for the reviewers ranks.
```

Comment:
I know 1:1 mentoring was mentioned below, but do have any sort of more formal guidelines for being added as a reviewer/approver to the aliases? Looking at the history of the group, it doesn't look like its changed much outside of removing an inactive member in 2 years =\